### ISSUE TITLE
Not able to install this using Node 18.17.1. Please help.

### ISSUE THREAD

**Issue by satya4satyanm** [Automattic/node-canvas#2303]
Not able to install this using Node 18.17.1. Please help.

**Comment by Miauwdy** [comment]
Same on Node 20.8.0.. I tried installing chartjs-node-canvas which relies on canvas so that one does not work either.

**Comment by alxwstn** [comment]
I wanted to call out that we ran into some issues recently with canvas install. Did y'all remove some older builds? This is link is now 404ing
https://github.com/Automattic/node-canvas/releases/download/v2.9.0/canvas-v2.9.0-node-v108-darwin-unknown-x64.tar.gz

We are using resemblejs v4.11 which expects canvas 2.9.0 as an optional dependency. We were able to overcome this issue using module resolutions in our package.json and pointing to 2.11.2. Hope this helps folks facing similar issues!
